doc/tutorials/transformer/masked_modelling.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<p>Maybe add a comment before this code block explaining this is calling the udf, and one after talking about the resut and g´finifhing of the tutorial. as it is now the end feels very abrupt.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>It does feel abrupt, I completely agree. Just don't know what to say.</p>

Reply:
<p>Something like " The result should look like &lt;explanation what it should look like&gt;. As you can see we successfully loaded and employed a filling-mask model using the Transformers Extension. If you want to see examples for the usage of other model types you an look here&lt;link to notebook overview&gt;" would work in my opinion.</p>